internal/pkg/manifest/lb_web_svc.go

```diff
@@ -202,3 +212,23 @@ func (s LoadBalancedWebService) ApplyEnv(envName string) (WorkloadManifest, erro
 	s.Environments = nil
 	return &s, nil
 }
+
+// WindowsCompatibility disallows unsupported services when deploying Windows containers on Fargate.
+func (s LoadBalancedWebService) WindowsCompatibility() error {
```

Comment:
wowy this is a tough one to keep DRY 😰 

I think we need to have a "parent" type:
```go
type ecsManifest struct {
  // Contains all the shared field across LoadBalancedWebService, BackendService, WorkerService
}
```
where we can put this method to. That would be the dream situation. For now, I'm okay getting the changes in this way, but I'd love to create a refactor PR into mainline where we embed `*ecsManifest` to existing types after this work.